main/zfs/APKBUILD

```diff
@@ -1,15 +1,15 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=zfs
-pkgver=0.7.12
-pkgrel=1
+pkgver=0.7.13
+pkgrel=0
 pkgdesc="ZFS for Linux"
 url="http://zfsonlinux.org"
 arch="all !armhf !armv7"
 license="CDDL"
 depends_dev="glib-dev e2fsprogs-dev util-linux-dev libtirpc-dev attr-dev"
 makedepends="$depends_dev automake autoconf libtool linux-headers"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-dracut::noarch
+subpackages="$pkgname-dev $pkgname-doc $pkgname-openrc $pkgname-libs $pkgname-dracut::noarch
 	$pkgname-udev $pkgname-scripts $pkgname-utils-py:utils_py:noarch"
 source="https://github.com/zfsonlinux/zfs/releases/download/zfs-$pkgver/zfs-$pkgver.tar.gz
 	zfs-import-should-be-before-swap.patch
```

Comment:
zfs-import-should-be-before-swap.patch is missing from sha512sums

Reply:
I have corrected the mistake.